indigo/distribution.yaml

```diff
@@ -10654,6 +10654,12 @@ repositories:
       url: https://github.com/intel-ros/realsense.git
       version: indigo-devel
     status: maintained
+  realsense_ros_camera:
+    doc:
+      type: git
+      url: https://github.com/intel-ros/realsense.git
```

Comment:
This looks to be a duplicate of: https://github.com/ros/rosdistro/pull/17045/files#diff-4bbb8e35ff5dc5e7f2d7b4d00374533fR10644 with a different branch. Do you want to change the documentation branch? If so then changing lin 10645 would be the right thing to do.